scrypto-unit/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "scrypto-unit"
+version = "0.1.0"
```

Comment:
Can we use the same version as other crates to avoid confusion?

Reply:
Indeed, will go ahead and correct the version :)